css/src/elementor.css

```diff
@@ -55,6 +57,9 @@
 
 .yoast a.dashicons {
```

Comment:
Somehow for this icon, in elementor it's not aligning with the bottom of the label (notice how it is a bit lower). Any way to get that aligned a bit nicer?

![image](https://user-images.githubusercontent.com/26220788/98355397-5ca18c00-2022-11eb-9e11-a6e8517f28d7.png)

Reply:
Wrote a fix for this but that seems to work inside the collapsible only. Will fix this.